systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture_thumbnail.url }}" class="profile-pic img-square img-responsive crop_img" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
+    <img src="{% static 'img/default.png' %}" class="profile-pic img-square img-responsive crop_img"
+    alt="{{ user }} profile picture"/>
+  </a>
+{% endif %}
+<a href="{{ user.get_absolute_url }}">
+  <p class="text-indent35"> {{ user|title }} </p>
```

Comment:
Regarding `user|title` - if I set my name as "ana", I want it to be "ana" and not "Ana". Be always careful when modifying any user input.